Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -22,20 +22,26 @@ namespace HoloToolkit.Unity
     public partial class GestureManager : Singleton<GestureManager>
     {
         /// <summary>
-        /// Occurs when a manipulation gesture has started
+        /// Occurs when a manipulation gesture has started.
         /// </summary>
-        public System.Action ManipulationStarted;
+        /// <param name="sourceKind">The Interaction Source Kind that started the event.</param>
+        public delegate void ManipulationStartedDelegate(InteractionSourceKind sourceKind);
+        public event ManipulationStartedDelegate OnManipulationStarted;
```

Comment:
I wonder about using UnityEvent instead of delegate/event - they make binding in both code and editor very simple.